deploy/concierge/schema.yml

```diff
@@ -0,0 +1,129 @@
+#! Copyright 2020-2021 the Pinniped contributors. All Rights Reserved.
+#! SPDX-License-Identifier: Apache-2.0
+
+#@data/values-schema
+---
+#@schema/desc "App name"
```

Comment:
I just added the description of fields everywhere as per my understanding please feel free to suggest a better description where you feel like it's not up to the mark.